catalog/util.sh

```diff
@@ -25,7 +25,7 @@ function createPackage() {
     else
         CMD_ARRAY=("$OPENWHISK_HOME/bin/go-cli/wsk" -i --apihost "$EDGE_HOST" package update --auth "$AUTH_KEY" --shared yes "$PACKAGE_NAME" "${REST[@]}")
     fi
-    export WSK_CONFIG_FILE= # override local property file to avoid namespace clashes
+    export WSK_CONFIG_FILE=None # override local property file to avoid namespace clashes
```

Comment:
@rabbah, I fixed the above logic to not use a config file if the WSK_CONFIG_FILE is set to an empty value.

For the test we could set the config file to a blank value then try to do a property set, which should fail since no property file is defined.